assets/src/components/OauthCallback.js

```diff
@@ -11,16 +11,36 @@ import { setToken } from '../helpers/auth'
 import { localized } from '../helpers/hostname'
 
 import { LoopingLogo } from './utils/AnimatedLogo'
+import { LoginPortal } from './Login'
 
 const CALLBACK = gql`
   mutation Callback($code: String!, $redirect: String) {
     oauthCallback(code: $code, redirect: $redirect) { jwt }
   }
 `
 
+function OauthError({ error: { error, error_description: description } }) {
+  return (
+    <LoginPortal>
+      <Box gap="medium">
+        <Box
+          gap="xsmall"
+          align="center"
+        >
+          <Alert
+            status={AlertStatus.ERROR}
+            header={error}
+            description={description || "You cannot log into this console instance, make sure your plural user was added to its oidc provider"}
+          />
+        </Box>
+      </Box>
+    </LoginPortal>
+  )
+}
+
```

Comment:
Wanna use some `<Div>` and/or `<Flex>` from honorable since we're trying to migrate off grommet?

Reply:
I think we need to rewrite this in the redesign anyway so will probably just punt for now